src/Apps/IdentityVerification/IdentityVerificationApp/StartIdentityVerification.tsx

```diff
@@ -0,0 +1,160 @@
+        },
+        onError: reject,
+        onCompleted: (response, errors) => {
+          if (errors && errors.length > 0) {
+            reject(new Error(JSON.stringify(errors)))
+          } else {
+            const {
+              startIdentityVerification: {
+                startIdentityVerificationResponseOrError: {
+                  identityVerificationFlowUrl,
+                  mutationError,
+                },
+              },
+            } = response
+            if (mutationError) {
+              reject(new Error(JSON.stringify(mutationError)))
+            } else {
+              resolve(identityVerificationFlowUrl)
+            }
+          }
+        },
+      })
+    })
+
+    mutation
+      .then(handleMutationSuccess)
+      .catch(handleMutationError)
+      .finally(() => {
+        setRequesting(false)
+      })
+  }
+
+  const handleMutationSuccess = (identityVerificationFlowUrl: string) => {
+    onContinueToVerification(identityVerificationFlowUrl)
+  }
+
+  const handleMutationError = (error: Error) => {
+    logger.error("Error when trying to start identity verification", error)
+    setShowErrorModal(true)
+  }
+  return (
+    <>
+      <ErrorModal
+        show={showErrorModal}
+        contactEmail="verification@artsy.net"
+        onClose={() => {
+          setShowErrorModal(false)
+        }}
+      />
+      <Box px={[2, 3]} mb={6} mt={4}>
+        <Box
+          mx="auto"
+          width={["100%", "80%"]}
+          maxWidth="400px"
+          textAlign="center"
+        >
+          <Serif size="6">Artsy identity verification</Serif>
+
+          <Box textAlign="left">
+            <Sans size="4" mt={2} weight="medium">
+              You’ll need
+            </Sans>
+            <Sans size="4">• A camera on your phone or computer</Sans>
+            <Sans size="4">• Your government ID </Sans>
+            <Sans size="4" mt={2} weight="medium">
+              Keep in mind
+            </Sans>
+            <Sans size="4">• Verification will take 5–10 minutes</Sans>
+            <Sans size="4">
+              • The name on your ID must match the name on your payment method
+            </Sans>
+            <Sans size="4">
+              • Your ID and photo will only be used for verification purposes,
+              and will not be shared
+            </Sans>
+            <br />
+            <Sans size="4">
+              By clicking the button, you'll be redirected to our identity
+              verification partner.
+            </Sans>
+          </Box>
+
+          <Button
+            loading={requesting}
+            block
+            width={["100%", 335]}
+            mt={4}
+            onClick={() => {
+              setRequesting(true)
+              trackClickedContinueToVerification()
+              startIdentityVerification()
+            }}
+          >
+            Continue to verification
+          </Button>
+        </Box>
+      </Box>
```

Comment:
I think we should bring back what is currently in the `<IdentityVerificationApp>` component. It seems like the old code was copy-&-pasted here, and it isn't just un-refactored code but also missing a link to FAQ.

https://github.com/yuki24/reaction/blob/bfffe3297b704f10378fa7d7831bac05427659da/src/Apps/IdentityVerification/IdentityVerificationApp.tsx#L117-L169

Reply:
Good catch! probably lost on rebase. I'll try restoring this version but personally don't like the mixture of inner components.

Reply:
replaced :+1: